cdap-standalone/bin/cdap.bat

```diff
@@ -205,6 +211,7 @@ SET "CLASSPATH=%CDAP_HOME%\lib\*;%CDAP_HOME%\conf\;"
 CALL :CHECK_WINDOWS
 CALL :CHECK_JAVA
 CALL :CREATE_LOG_DIR

```

Comment:
I would think so, but would be unrelated to this PR. I can fix it in this PR. @wolf31o2 any thoughts?